feeds/trilliumtransit.com.dmfr.json

```diff
@@ -3278,6 +3357,20 @@
         }
       ]
     },
+    {
+      "id": "f-cityoflawndale~ca~us",
```

Comment:
no need for a new feed record. we have https://www.transit.land/feeds/f-9q5bf-lawndale~ca~us

@westontrillium, is there any meaningful difference between https://ridelawndalebeat.com/gtfs and https://data.trilliumtransit.com/gtfs/cityoflawndale-ca-us/cityoflawndale-ca-us.zip ?

Reply:
Similar situation as Chapel Hill except in this case Trillium's feed _is_ the source of truth and is being ingested by consumers.

Reply:
@westontrillium it looks like the current contents of https://ridelawndalebeat.com/gtfs and https://data.trilliumtransit.com/gtfs/cityoflawndale-ca-us/cityoflawndale-ca-us.zip are quite different. Different enough that the Trillium feed would not pair well with the Syncromatics RT endpoints.

For now, let's just use the Syncromatics feed, so we don't have conflicts.

But feel free to let us know more by email if there's more context about how Trillium and Syncromatics feeds relate in general.